tests/src/python/CMakeLists.txt

```
@@ -405,7 +406,7 @@ if (ENABLE_PGTEST)
     PyQgsRelationEditWidget PyQgsRelationPostgres PyQgsVectorLayerTools PyQgsProjectStoragePostgres
     PyQgsAuthManagerPKIPostgresTest PyQgsAuthManagerPasswordPostgresTest PyQgsAuthManagerOgrPostgresTest
     PyQgsDbManagerPostgis PyQgsDatabaseSchemaModel PyQgsDatabaseTableModel PyQgsDatabaseSchemaComboBox PyQgsDatabaseTableComboBox
-    PyQgsProviderConnectionPostgres
+    PyQgsProviderConnectionPostgres PyQgsPostgresProviderRemote
```

```suggestion
    PyQgsProviderConnectionPostgres PyQgsPostgresProviderLatency
```

I think that makes the purpose a bit clearer.
Otherwise good to go